### ISSUE TITLE
Error running aurora inside of omfit

### ISSUE THREAD

**Issue by jmcclena** [fsciortino/Aurora#12]
@fsciortino @orso82  I get this error trying to import aurora inside of omfit:

```Error in "OMFIT command box #2" at line  1
    import aurora
NameError: name 'SortedDict' is not defined


Press <F6> to see full error report...

Error in "OMFIT command box #2" at line  1
    import aurora
NameError: name 'SortedDict' is not defined


Press <F6> to see full error report...

Traceback (most recent call last):
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/OMFITx.py", line 4071, in manage_user_errors
    tmp=command(**kw)
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/utils_tk.py", line 1060, in GlobLoc_tk
    return py.run(_relLoc=self.namespace, _OMFITscriptsDict=False, _OMFITconsoleDict=True, noGUI=None)
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/omfit_python.py", line 1082, in run
    result = self.__run__(**kw)
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/omfit_python.py", line 931, in __run__
    self, userDict=kw, inputDict=_relLoc, persistentDict=OMFITconsoleDict, runDict={}, prerun=prerun, postrun=postrun
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/omfit_python.py", line 129, in f_locked
    return f(*args, **kw)
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/omfit_python.py", line 362, in execGlobLoc
    exec(compile(execString, filename, "exec"), GlobLoc)
  File "/tmp/mcclenaghan/OMFIT/OMFIT_2020-11-24_08_53_06_787689/project/OMFIT command box #2", line 1, in <module>
    import aurora
  File "/Users/mcclenaghan/programming/Aurora/aurora/__init__.py", line 11, in <module>
    from .core import *
  File "/Users/mcclenaghan/programming/Aurora/aurora/core.py", line 13, in <module>
    import omfit_eqdsk
  File "/Users/mcclenaghan/programming/OMFIT-source/omfit/classes/omfit_eqdsk.py", line 227, in <module>
    class OMFITgeqdsk(SortedDict, OMFITascii):
NameError: name 'SortedDict' is not defined```

**Comment by jmcclena** [comment]
I'm sure its not the right solution, but if I add the following lines to my omfit source, I can get aurora to import:

to omfit_mds.py:
from classes.sortedDict import *
from classes.utils_base import *

to omfit_eqdsk.py:
from classes.sortedDict import *

**Comment by orso82** [comment]
@jmcclena I would assume that when you `pip install aurora` the dependencies of aurora on the OMFIT-classes were installed. Can you please try to start an fresh python session at the terminal (using the same Python that OMFIT uses) and then do an `import aurora` there. What do you get?

**Comment by jmcclena** [comment]
@orso82 I installed aurora with the setup script (perhaps what led to my problems), as that was the recommended method for installation. @fsciortino @StanczakDominik  is it still the preferred installation method? So perhaps the omfit source location needs to be set somewhere in the setup script?

**Comment by fsciortino** [comment]
@jmcclena using the makefile in the Aurora distribution directory with
```
make clean; make aurora
```
will set things up, but won't install any of the requirements, which you'd need to install one by one by yourself (there isn't many). PR #11 is on improving the installation and tests procedures (@StanczakDominik is showing me some awesome tricks there) and we'll probably merge soon. 

@orso82 the issue seems to be a missing import statement for `SortedDict` in `omfit_eqdsk`. I think that there should be an explicit `from omfit_commonclasses import SortedDict`, or something similar. Could you please check?

**Comment by StanczakDominik** [comment]
Darn. Yeah, this is on me: in #12 I suggested either python setup.py develop or pip install --editable after having broken stuff in an earlier PR; but forgot setup.py wouldn't install the dependencies. Apologies!

**Comment by fsciortino** [comment]
@orso82 quick reminder about my comment above, suggesting that an import of `SortedDict` might be missing from `omfit_eqdsk`.

**Comment by orso82** [comment]
@fsciortino there `SortedDict` class is imported as part of the `from .startup_choice import *` statement at the beginning of the `omfit_eqdsk.py` file. The issue here is how `omfit_eqdsk` loaded by OMFIT and `aurora` are playing with one another.

https://github.com/gafusion/OMFIT-source/pull/5089 should fix it

**Comment by jmcclena** [comment]
fixed with gafusion/OMFIT-source#5089
